blocks/hero/hero.css

```diff
@@ -101,7 +105,7 @@ udex-hero-banner#media-blend .media-blend__intro-text {
   color: var(--udexColorGrey6, #5b738b);
 }
 
-udex-hero-banner#media-blend .media-blend__content h1 {
+udex-hero-banner .hero-banner .media-blend__content h1 {
```

Comment:
@mhaack Since `appendChild` is replaced with append in line 61 of` hero.js`  `.hero-banner `class is at same level as `.media-blend__content` so these rules no longer apply. Same is the case with `hero-banner media-blend__additional-content`. We should remove these rules.

Reply:
I can also clean up the CSS as part of #220. Let me know whats best.

Reply:
fixed